src/_includes/head.hbs

```diff
@@ -0,0 +1,38 @@
+<meta charset="utf-8">
+<meta http-equiv="X-UA-Compatible" content="IE=edge">
+<meta http-equiv="Content-Type" content="text/html; charset=utf-8">
+<meta name="viewport" content="width=device-width, initial-scale=1">
+
+<title>{{htmlWebpackPlugin.options.site.title}}</title>
+<meta name="description" content="{{htmlWebpackPlugin.options.site.description}}">
+
+<link rel="canonical" href="">
```

Comment:
Looks like this got skipped by accident

Reply:
after looking at various mlh sites canonical seems to be the current address you're on, for example when you go to `https://careerlab.mlh.io/partners/carfax/`, the same url would be an href in the link. Without Jekyll the only way I found to populate this field is with javascript(but might be too late, not really sure how it works).